scripts/release/sign.sh

```diff
@@ -24,18 +24,18 @@ fi
 
 # TODO: add gateway-cln-extension once available as an output
 for bin in fedimintd fedimint-cli fedimint-dbtool gateway-cli gatewayd ; do
-  repo="git+file:${REPO_ROOT}?ref=refs/tags/${tag}"
+  # We need to use rev= , see https://github.com/NixOS/nix/issues/11266
+  # NOTE: '^{commit}' is not a mistake, but git rev-parse syntaxh
```

Comment:
[nit] s/syntaxh/syntax

---

That's a nifty trick :+1: 

For others that are curious

```
       o   Print the commit object name from the revision in the $REV shell
           variable:

               $ git rev-parse --verify --end-of-options $REV^{commit}

           This will error out if $REV is empty or not a valid revision.
```